ui/src/containers/NodeVolumes.js

```diff
@@ -77,8 +179,85 @@ const NodeVolumes = props => {
           />
         </span>
       )
+    },
+    {
+      label: intl.messages.action,
+      dataKey: 'action',
+      disableSort: true,
+      width: 80,
+      renderer: (data, rowData) => {
+        const isEnableClick = isVolumeDeletable(rowData);
+        const isTriggerTooltip = () => {
+          if (isEnableClick) {
+            return '';
+          } else {
+            return 'hover';
+          }
+        };
+
+        const hintPopup = () => {
+          let hintMessage = '';
+
+          switch (rowData.status) {
+            case STATUS_PENDING:
+              hintMessage = intl.messages.volume_status_pending_hint;
+              break;
+            case STATUS_TERMINATING:
+              hintMessage = intl.messages.volume_status_terminating_hint;
+              break;
+            case STATUS_UNKNOWN:
+              hintMessage = intl.messages.volume_status_unknown_hint;
+              break;
+            case STATUS_FAILED:
```

Comment:
This should be:

```
case STATUS_FAILED:
case STATUS_AVAILABLE:
```

Otherwise you'll fall into the `default` case l.229 and will have the behavior I noticed (see my message on our Slack channel).